### PR TITLE
Multiple field relations

### LINKED ISSUE #4087: Allow to define multiple field pairs for relations (composite keys)
Original commit: https://github.com/qgis/QGIS/commit/9b4cb6f502c4f01641e0f0f7dd2ff3ac232c110f by web-flow

[FEATURE] allow to define multiple field pairs for relations (composite keys)

### PR DESCRIPTION
**Pull request by lajunek**
Goal:Users shall be able to use multiple field pairs for relations.

Ticket(s): fix #4087

- [x ] Backport to LTR documentation is required

### CODE REVIEW

**Comment by lajunek** [comment]
@hatav sphinx doesn't build here. How can I solve the problem?

File: docs/user_manual/working_with_vector/attribute_table.rst [hunk 1]
```diff
@@ -691,12 +691,13 @@ Open the :guilabel:`Relations` tab and click on :guilabel:`Add Relation`.
   describing, what the relation is used for. We will just call say **Airports**
   in this case.
 * **Referenced Layer (Parent)** also considered as parent layer, is the one with
-  the primary key, pointed to, so here it is the ``regions`` layer
-* **Referenced Field** is the primary key of the referenced layer so it is ``ID``
+  the primary key, pointed to, so here it is the ``regions`` layer. For this layer
+  you can define multiple referenced fields as primary key, so these are ``ID``
+  and ``a_airports.
```
Comment (havatv) [inline docs/user_manual/working_with_vector/attribute_table.rst:696]:
```suggestion
  and ``a_airports``.
```

**Comment by havatv** [comment]
> @hatav sphinx doesn't build here. How can I solve the problem?

I found a place with missing

    ``

That may be the cause of the problems. I have made a suggestion to correct this.

File: docs/user_manual/working_with_vector/attribute_table.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (havatv) [reply docs/user_manual/working_with_vector/attribute_table.rst]: I had already committed this, and it helped.

**Comment by lajunek** [comment]
@DelazJ @havatv during the backport process there was the following message: Unexpected input(s) 'bot_token_key', valid inputs are ['bot_username', 'bot_token', 'github_token']. Actually I can't find my changes in the user manual website

**Comment by havatv** [comment]
> @DelazJ @havatv during the backport process there was the following message: Unexpected input(s) 'bot_token_key', valid inputs are ['bot_username', 'bot_token', 'github_token']. Actually I can't find my changes in the user manual website

Not sure what you are talking about.  Are you doing a manual backport? I added the backport label. The "automatic" backport says "no conflicts", so seems to be OK.

**Comment by DelazJ** [comment]
>  The "automatic" backport says "no conflicts", so seems to be OK.

"No conflicts" but it does not pick any screenshots. green travis does not necessarily mean it's OK. Fixed in #5913

**Pull request merged**